staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/admission_test.go

```diff
@@ -460,6 +468,38 @@ func must3[T any, I any](val T, _ I, err error) T {
 // Functionality Tests
 ////////////////////////////////////////////////////////////////////////////////
 
+func TestPluginNotReady(t *testing.T) {
```

Comment:
this test takes 10s to complete due to:

https://github.com/kubernetes/kubernetes/blob/76b4699c692972355267b05b2326d881210bb660/staging/src/k8s.io/apiserver/pkg/admission/handler.go#L26-L28

I'm not sure if it is best to include it or selectively enable it, etc. Or is it ok because it is run in parallel with many others?

Reply:
I guess that is why we don't have much admission_test do this check in the code :)

Reply:
You could create a new `timeout` field in handler that defaults to `timeToWaitForReady`, and a new method `SetTimeout()`  to override it from tests.

Reply:
I agree that is probably ideal. I was hoping to keep the change as low impact as possible due to code freeze/approvals. I wonder if that is something the release managers might allow to go in past code freeze?

Reply:
It is ok to update with suggested fix if you like since it is considered as a fix plus reviewed/approved before code freeze.